lib/cocina/generator/datatype.rb

```diff
@@ -10,13 +10,23 @@ def generate
 
           module Cocina
             module Models
-              #{name} = Types::String.constrained(
-                format: /#{schema_doc.pattern}/
-              )
```

Comment:
Trying to get the generator out of the business of formatting model code, letting Rubocop do its thing.